lib/annotated/init.go

```diff
@@ -431,3 +431,18 @@ func (cs *acs) deploy(ctx context.Context) (err error) {
 
 	return nil
 }
+
+func (cs *acs) Reload() error {
+	// hope this is good if it worked before.
+	connectString, _ := os.LookupEnv(envVarConn)
```

Comment:
You can just pull it back from oldPool: https://godoc.org/github.com/jackc/pgx/pgxpool#Config.ConnString